pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -421,6 +422,22 @@ func (cm *containerManagerImpl) PodHasExclusiveCPUs(pod *v1.Pod) bool {
 }
 
 func (cm *containerManagerImpl) ContainerHasExclusiveCPUs(pod *v1.Pod, container *v1.Container) bool {
+	if utilfeature.DefaultFeatureGate.Enabled(kubefeatures.PodLevelResourceManagers) && resourcehelper.IsPodLevelResourcesSet(pod) {
+		// With pod-level resource management, a container is considered to have exclusive CPUs
+		// if it is in a Guaranteed pod, requests an integer number of CPUs and specifies an
+		// equal request and limit amout for both CPU and memory.
```

Comment:
nit: isn't this redundant with the Guaranteed pod requirement?

> and specifies an equal request and limit amout for both CPU and memory.

Reply:
Using pod-level resources, you can have a Guaranteed pod, with best effort containers. This is why we need to still make this check.